src/Bathymetry.jl

```diff
@@ -135,6 +148,38 @@ function regrid_bathymetry(target_grid; dataset = ETOPO2022(), kw...)
     return regrid_bathymetry(target_grid, metadatum; kw...)
 end
 
+# Regridding bathymetry for distributed grids, we handle the whole process
+# on just one rank, and share the results with the other processors.
+function regrid_bathymetry(target_grid::DistributedGrid, args...; kw...)
```

Comment:
```suggestion
function regrid_bathymetry(target_grid::DistributedGrid, metadatum; kw...)
    download_dataset(metadatum)
```

Reply:
I left it as `args...` so that the convenience method with the default dataset also works.

Reply:
nice catch; thanks!